packages/react-components/src/molecules/__tests__/SearchField.test.tsx

```diff
@@ -35,6 +35,25 @@ describe('GTM data', () => {
       expect.anything(),
     );
   });
+  it('shows an custom X to clear the query', () => {
+    const { getByRole } = render(<SearchField placeholder="p" value="q" />);
```

Comment:
This test should be improved. You should use rerender. First without a value and ensuring the button isn't present. The second rerender with the value and the button is present